presentation/build.gradle

```diff
@@ -11,6 +11,7 @@ android {
         buildConfigField "boolean", "ENABLE_LINK_APPERANCE_MENU", "true"
         buildConfigField "boolean", "USE_SIMPLE_TABLES_IN_EDITOR_EDDITING", "true"
         buildConfigField "boolean", "ENABLE_WIDGETS", "false"
+        buildConfigField "boolean", "ENABLE_VIEWS_MENU", "false"
```

Comment:
please consider more specific naming - what kind of menu?

Reply:
same kind of view's settings menus with updated design etc